common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -193,6 +230,12 @@ def create_response_element(self, **kwargs):
             responseparam_element = etree.SubElement(response_element, 'responseparam')
             responseparam_element.set('type', 'tolerance')
             responseparam_element.set('default', str(tolerance))
+            if partial_range is not None and 'close' in credit_type:
+                responseparam_element.set('partial_range', str(partial_range))
+
+        if partial_answers is not None and 'list' in credit_type:
+            responseparam_element = etree.SubElement(response_element, 'responseparam')
```

Comment:
Line 237 here is throwing a pylint error. Line 230 is not. I have no idea why, since I literally copy-pasted this one from that one. Any ideas?

The error is `237: E1101: (no-member), NumericalResponseXMLFactory.create_response_element: Module 'safe_lxml.etree' has no 'SubElement' member`